JavaScript/Merge_pdfs/README.md

```diff
@@ -0,0 +1,34 @@
+# Merging two pdf files
+
```

Comment:
Can you describe the use-case of this particular script in more detail in the README?